### PR TITLE
added footer

### PR DESCRIPTION
**Pull request by 0xMALVEE**
😊Done list: 
1. Added footer
2. Tailwind css

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,10 +3,20 @@
...
+    <script src="https://cdn.tailwindcss.com"></script>
   </head>
   <body>
     <h1>Hello webpack!</h1>
     <h2>this is test again right now?</h2>
+
+    <footer class="bg-gray-800 text-white py-4 text-center">
+      <div class="container mx-auto">
+          <p class="text-sm">
+              Created by <strong class="font-bold">Alvee</strong>, <strong class="font-bold">Esteban</strong>, and <strong class="font-bold">Daniel</strong> under MIT License
```
Comment (Estete9) [inline src/index.html:16]: - [ ] kindly change the third name to Alex 😄

**Review by Estete9 — CHANGES_REQUESTED** [review]
This footer looks great Alvee, thank you for your help 💪 
Help me with one thing.

**Comment by 0xMALVEE** [comment]
Changes Made,
Changed name to Alex.

**Review by Estete9 — APPROVED** [review]
STATUS: APPROVED 🟢 ✔️

Thank you for your help 🥳

**Pull request merged**